Sources/xcproj/PBXProj+Helpers.swift

```diff
@@ -22,17 +22,10 @@ extension PBXProj {
     /// - Parameter fileReference: file reference.
     /// - Returns: file name.
     func fileName(fileReference: String) -> String? {
-        if let variantGroup: PBXVariantGroup = objects.variantGroups.getReference(fileReference) {
-            return variantGroup.name
-        } else if let group: PBXGroup = objects.groups.getReference(fileReference) {
-            return group.name
-        } else if let versionGroup: XCVersionGroup = objects.versionGroups.getReference(fileReference) {
-            return versionGroup.name
-        } else if let fileReference: PBXFileReference = objects.fileReferences.getReference(fileReference) {
-            return fileReference.name ?? fileReference.path
-        } else {
+        guard let fileElement = objects.getFileElement(reference: fileReference) else {
             return nil
         }
+        return fileElement.name ?? fileElement.path
```

Comment:
Will this cause any issues? Before only the fileReference was returning the path if no name was present. If so it can be changed @pepibumur

Reply:
AFAIK it shouldn't @yonaskolb. I think groups, variant groups and version groups they should all have a name so the path wouldn't be returned in those cases.